src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -110,89 +108,96 @@ define(function (require, exports, module) {
         _handleLibraryRemove: function () {
             this.getFlux().actions.libraries.removeCurrentLibrary();
         },
-        
+
         /**
          * Return library panel content based on the connection status of CC Library.
          * @private
-         * 
+         *
          * @return {ReactComponent}
          */
         _renderLibrariesContent: function () {
+            if (!this.props.visible || this.props.disabled) {
```

Comment:
thanks for bringing this up. the rendering function has control over the output, so I think it is reasonable for it to rendering nothing, or an empty block (to fit with its parent). I am also thinking, if remove this, the checking here will throw back to the `render`, which will make it become more complicate unnecessarily. I will update the `@return` to make it more clear.